src/ua.skrypnyk.hw10/Square.java

```diff
@@ -0,0 +1,35 @@
+package src.ua.skrypnyk.hw10;
+
+import java.util.Scanner;
+
+//Perimeter and area should be calculated
+public class Square extends Shape{
+    private double perimeter;
+    private double area;
+    private double side;
+
+    public Square(double side) {
+        this.side = side;
+        }
+
+    @Override
+    public double area() {
+        try {
+
+            if (side <= 0) {
+                throw new isAreaOfShapeValid("Side should be bigger null");
```

Comment:
Please, use more propriate name for the exception:
For example: InvalidDataException